.github/workflows/knip.yml

```diff
@@ -0,0 +1,28 @@
+name: Knip
+
+on:
+  pull_request:
+
+permissions: write-all
+
+jobs:
+  run-knip:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Setup Node
+        uses: actions/setup-node@v3
+        with:
+          node-version: "18.14.1"
+
+      - name: Install toolchain
+        run: yarn install
+
+      - name: Report knip results to pull request
+        uses: gitcoindev/knip-reporter@v1.2.9
```

Comment:
Perhaps its best to place this under Ubiquity?

Reply:
Ok let me check the update.

Reply:
Let's wait a few days for the upstream pull request to be merged.  If it will stall we'll move under Ubiquity, but since the author released knip-reporter with v3 support this week there is a high chance he will pick it up soon.

Reply:
Okay keep me posted!